pkg/oc/cli/admin/mustgather/mustgather.go

```diff
@@ -142,24 +146,40 @@ func (o *MustGatherOptions) Run(rsyncCmd *cobra.Command) error {
 	if err != nil {
 		return err
 	}
+	o.PrinterCreated.PrintObj(ns, o.Out)
 	if !o.Keep {
 		defer func() {
-			err = o.Client.CoreV1().Namespaces().Delete(ns.Name, nil)
+			err2 := o.Client.CoreV1().Namespaces().Delete(ns.Name, nil)
+			if err2 != nil {
+				if err != nil {
+					err = fmt.Errorf("%v\n%v", err, err2)
```

Comment:
yeah... don't do this.  use a channel if we need a channel.  For now, just suppress the error code and fmt.Printf() it out.